docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -121,26 +127,32 @@ endif::[]
 
 Before you use {product-connectors}, you must complete the following prerequisites: 
 
-* Determine which {openshift} environment to use for deploying your {product-connectors} instances.
+* Determine which {openshift} environment to use for your _{connectors} namespace_. The {connectors} namespace is where your {product-connectors} instances will be deployed.
 
 * Configure {product-long-kafka} for use with {product-connectors}.
 
-*Determining which {openshift} environment to use for deploying your {connectors} instances*
+*Determining which {openshift} environment to use for your {connectors} namespace*
 
-For Service Preview, you have two choices:
+You have three choices:
 
 * *The hosted preview environment*
 
 ** The {connectors} instances are hosted on a multitenant {osd-name-short} cluster that is owned by Red Hat.
```

Comment:
Perhaps rename the attribute from `osd-name-short` to `product-osd`?
You might want to create a `product-long-osd` attribute too, for the first occurrence?

Reply:
My apologies, I steered you in the wrong direction with this suggestion. :(

As discussed on chat, we probably don't need two attributes for this product, especially now that we have confirmation that the short form "OSD" is not approved.